### PR TITLE
002 view job role info

### PR DESCRIPTION
**Pull request by AZZohaib**

### CODE REVIEW

File: src/main/java/org/example/controllers/JobRoleController.java [hunk 1]
```diff
@@ -1,29 +1,58 @@
...
+            return Response.serverError().build();
+        }
+    }
+
     @GET
     @Produces(MediaType.APPLICATION_JSON)
-    public Response getJobRoles() throws SQLException {
+    @ApiOperation(
+            value = "Returns all Job Roles",
+            authorizations = @Authorization(value = HttpHeaders.AUTHORIZATION),
```
Comment (JennicaMistry) [inline src/main/java/org/example/controllers/JobRoleController.java]: Remove please

File: src/main/java/org/example/models/JobRoleInfo.java [hunk 2]
```diff
@@ -0,0 +1,118 @@
+package org.example.models;
+
```
Comment (JennicaMistry) [inline src/main/java/org/example/models/JobRoleInfo.java]: Name of the file needs to have response.

**Review by JennicaMistry — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/controllers/JobRoleController.java (hunk 1 above)
Comment (AZZohaib) [reply src/main/java/org/example/controllers/JobRoleController.java]: Done

File: src/main/java/org/example/models/JobRoleInfo.java (hunk 2 above)
Comment (AZZohaib) [reply src/main/java/org/example/models/JobRoleInfo.java]: Done

**Review by JennicaMistry — APPROVED** [review]

**Review by Adnanhoq — APPROVED** [review]

**Pull request merged**